backend/app/api/src/endpoints/admin/organizations/SearchUitpasEventsEndpoint.ts

```diff
@@ -0,0 +1,51 @@
+import { DecodedRequest, Endpoint, Request, Response } from '@simonbackx/simple-endpoints';
+import { UitpasService } from '../../../services/uitpas/UitpasService';
+import { AutoEncoder, Decoder, field, StringDecoder } from '@simonbackx/simple-encoding';
+import { Context } from '../../../helpers/Context';
+import { UitpasEventsResponse } from '@stamhoofd/structures';
+import { SimpleError } from '@simonbackx/simple-errors';
+
+type Params = Record<string, never>;
+class Query extends AutoEncoder {
+    @field({ decoder: StringDecoder })
+    text: string;
+}
+type Body = undefined;
+type ResponseBody = UitpasEventsResponse;
+
+export class SearchUitpasEventsEndpoint extends Endpoint<Params, Query, Body, ResponseBody> {
```

Comment:
Te verplaatsen naar `backend/app/api/src/endpoints/organization/dashboard/webshops` (`admin` folder is voor administratieportaal)